src/components/parent/Inventory.vue

```diff
@@ -75,13 +108,27 @@ export default defineComponent({
 
     const dummyEquippedList = [dummyItemB, dummyItemA];
 
+    const equipped = computed<IBasePart[]>(() => props.parts.filter((it) => isSlotEquipped(it)));
+
+    const navigateToChild = (id: string): void => {
```

Comment:
Hi @bobo-k2 
For the equipped items, I assume `tokenId` for the "/child page" can be found in `z`. Please let me know if this is not correct.
![image](https://user-images.githubusercontent.com/92044428/223052085-22a3d540-b8a9-462e-a382-09f625e76cb9.png)

Reply:
![image](https://user-images.githubusercontent.com/92044428/223052500-cd77d9c2-1923-4c37-a93a-82a6ee79784c.png)

Reply:
Hi @impelcrypto this is not correct. Z variable determines in which layer asset should be rendered. Something like z-index in css.
To get token id yo ushould use `equippable::get_equipment`. The method, beside others, returns equipped item contract address and token id.